go/test/endtoend/onlineddl/vrepl_stress_suite/onlineddl_vrepl_stress_suite_test.go

```diff
@@ -697,9 +703,27 @@ func runSingleConnection(ctx context.Context, t *testing.T, autoIncInsert bool,
 				// Table renamed to _before, due to -vreplication-test-suite flag
 				err = nil
 			}
+			if sqlErr, ok := err.(*mysql.SQLError); ok {
+				switch sqlErr.Number() {
+				case mysql.ERLockDeadlock:
+					// That's fine. We create a lot of contention; some transactions will deadlock and
+					// rollback. It happens, and we can ignore those and keep on going.
+					err = nil
+				}
+			}
 		}
 		assert.Nil(t, err)
 		time.Sleep(singleConnectionSleepInterval)
+		// Most o fthe time, we want the load to be high, so as to create real stress and potentially
```

Comment:
```suggestion
		// Most of the time, we want the load to be high, so as to create real stress and potentially
```

Reply:
Whoops. Merged without noticing this comment.